README.md

```diff
@@ -150,6 +150,25 @@ A wrapper is provided with pipenv (look in the Pipfile's `[scripts]` block for m
 
 **NB**: Make sure to be in the virtual environment (`pipenv shell`) before running the tests.
 
+### Running tests with docker
+
+If you are unable to run tests locally because of pyodbc you can use the docker-compose
+
+        docker-compose up
+
+2. you will need to run the sqlserver with:
```

Comment:
This just creates the database, the SQLServer is running since it's a service in the compose file.

Reply:
Did you want to update this comment?